pkg/portforward/portforward.go

```diff
@@ -49,9 +49,10 @@ type PortForward interface {
 // portForwarder implements PortForward interface
 type portForwarder struct {
 	genericclioptions.RESTClientGetter
-	ctx    context.Context
-	config *rest.Config
-	client kubernetes.Interface
+	ctx       context.Context
+	config    *rest.Config
+	client    kubernetes.Interface
+	enableLog bool
```

Comment:
I think we do not need `enableLog`, more logs is not a bad thing.

Reply:
Hi @STRRL the thing is somewhat like this. By default, everytime we run `portforward`, one line of log would be print to show the pod and the port. I don't know if it would be helpful for debugging e2e test but I still keep it there.
https://github.com/chaos-mesh/chaos-mesh/blob/478d00d01bb0f9fb08a1085428a7da8c8f9df4e8/test/e2e/util/portforward/portforward.go#L80-L85

Considering we would need the same function for `chaosctl`, it could be kind of misleading for users if several lines of this kind of log print out. Since there is no something like a switch to disable `klog` output, I add this flag to disable the output when needed.

Reply:
Oh, you are right. 🙈

Reply:
Cool 😁